stash/README.md

```diff
@@ -45,7 +45,7 @@ To restore a stash before your build process, use the `infrastructure-actions/st
 ```yaml
 steps:
 - uses: actions/checkout@v2
-- uses: infrastructure-actions/stash/restore@v1
+- uses: apache/infrastructure-actions/stash/restore@main
```

Comment:
Might be a good idea to mention a commit hash - this is really the only way we **should** be using actions (and `zizmor` action and "CodeQL` security check will flag both tag an branch usage.

I think it would be better to follow this in the example.